src/coreclr/jit/simdcodegenxarch.cpp

```diff
@@ -285,11 +285,11 @@ void CodeGen::genPutArgStkSIMD12(GenTree* treeNode)
 #endif // TARGET_X86
 
```

Comment:
`simdcodegenxarch.cpp` is now "unique" and should probably just be merged into `codegenxarch.cpp`, just like the related code already is for Arm64.